docs/languages/cpp.md

```diff
@@ -182,10 +176,10 @@ The C/C++ extension for Visual Studio Code supports source code formatting using
 
 You can format an entire file with **Format Document** (`kb(editor.action.formatDocument)`) or just the current selection with **Format Selection** (`kb(editor.action.formatSelection)`) in right-click context menu. You can also configure auto-formatting with the following [settings](/docs/getstarted/settings.md):
```

Comment:
This section seems to be talking about code formatting, but the title says building?

Reply:
I'm not seeing what you're seeing.  This is directly under the "Editing Code" -> "Code Formatting" section.

Reply:
Oh, I see it now.  GitHub's diff view cuts out a lots of unchanged lines.  It's really confusing, but you can see the jump in line numbers in the left column.